book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
Good job so far, but you're missing out on:

- [ ] A method for adding rentals. It's required that you have a method for adding rental in your book class. Kindly work on that 👍